ui/packages/catalog-ui-search/src/main/webapp/component/search-form-selector/search-form-selector.view.js

```diff
@@ -31,8 +31,6 @@ module.exports = Marionette.LayoutView.extend({
     this.tabsContent.show(
       new SearchFormsView({
         queryModel: this.model,
```

Comment:
Do we still use this component at all?  I'm wondering if we can delete this file now.

Reply:
Wow, good call. Removed.